react/features/filmstrip/components/native/LocalThumbnail.js

```diff
@@ -32,7 +32,9 @@ class LocalThumbnail extends Component<Props> {
 
         return (
             <View style = { styles.localThumbnail }>
-                <Thumbnail participant = { _localParticipant } />
+                <Thumbnail
+                    participant = { _localParticipant }
+                    zOrder = { 2 } />
```

Comment:
Right, 0 is the bottommost layer in which the large video resides, 1 is the layer where the remote thumbnails are and that leaves 2 to the local thumbnail. Unfortunately and that's the reason why I never utilized 2, 2 means "above everything" which (in our case means) that we cannot render anything on top of it but we want to render indicators such as moderator, audio and video muted (and in the redesigned UI we'll want the "Toggle camera" button on top of it).